service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AllocatedJudgeContentProvider.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.notify.AllocatedJudgeTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.AbstractEmailContentProvider;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class AllocatedJudgeContentProvider extends AbstractEmailContentProvider {
+
+    private final ObjectMapper mapper;
+
+    public AllocatedJudgeTemplate buildAllocatedJudgeNotificationParameters(CaseDetails caseDetails) {
```

Comment:
I feel like this name is overly verbose, from the class name we know that we are going to be using functions that are related to allocated judge so I would probs just call this something like buildNotificationParameters, I feel like this provides all the info necessary without being too wordy

Reply:
cool makes sense :)